src/apiCalls.tsx

```diff
@@ -5,7 +5,7 @@ export const getWeddingInfo = (weddingId:number) => {
 }
 
 export const getWeddings = () => {
-  return fetch('http://localhost:8000/api/v1/weddings/weddings/')
+  return fetch("http://matrimania-backend.herokuapp.com/api/v1/weddings/weddings/")
```

Comment:
We have a deployed back end!!! Nice work!

Reply:
Thank you @IamNorma and @fieldstyler!!